### PR TITLE
refactor: small cleanup

### PR DESCRIPTION
**Pull request by 0xClandestine**
### Changes
- removes unused `ISignatureUtils` import for `IAllocationManager`.
- add missing `InputArrayLengthMismatch()` check to `slashOperator()`.
- fix `getQueuedWithdrawals` bug + test

### CODE REVIEW

File: src/test/unit/DelegationUnit.t.sol [hunk 1]
```diff
@@ -8274,3 +8273,56 @@ contract DelegationManagerUnitTests_Lifecycle is DelegationManagerUnitTests {
...
+            staker: defaultStaker,
+            withdrawer: defaultStaker,
+            strategies: strategies,
+            depositWithdrawalAmounts: depositShares
+        });
+
+        cheats.prank(defaultStaker);
+        delegationManager.queueWithdrawals(queuedWithdrawalParams);
+
+        // Get queued withdrawals.
```
Comment (ypatil12) [inline src/test/unit/DelegationUnit.t.sol:8317]: As a regression test, the error we had is when `i > j`. Therefore, we want the length of `totalQueued` to be greater than the number of strategies in a withdrawal. In this case its equal

Comment (ypatil12) [reply src/test/unit/DelegationUnit.t.sol:8317]: This could be an additional test to the correctness one we have here

**Review by ypatil12 — COMMENTED** [review]
Looks good, comment on the regression test

**Review by 8sunyuan — APPROVED** [review]
LGTM

**Pull request merged**